### PR TITLE
docs: clarify mouse binds

### PR DESCRIPTION
**Pull request by stefonarch**
Fixes https://github.com/labwc/labwc/issues/1075
Not sure if also scroll action can be combined, didn't test.

### CODE REVIEW

File: docs/labwc-config.5.scd [hunk 1]
```diff
@@ -344,7 +344,7 @@ Therefore, where multiple objects of the same kind are required (for example
 	- Desktop: The desktop background, where no windows are present.
 	- Root: A synonym for Desktop (for compatibility).
 
-	Supported mouse *buttons* are:
+	Supported mouse *buttons* can be combined with modifiers and are:
```
Comment (Consolatis) [inline docs/labwc-config.5.scd:347]: Should we say `keyboard modifiers` here to make it clear what 'modifiers' means in this context?

**Comment by Consolatis** [comment]
Thanks. While looking through the diff I also recognized in the section below which list the supported actions, the `action` word is not enclosed in `*` like `*button*` and `*direction*` above. Would you mind changing that as well?

Short comment about the commit title: Maybe consider something like `docs: clarify mouse binds`.

File: docs/labwc-config.5.scd (hunk 1 above)
Comment (johanmalm) [reply docs/labwc-config.5.scd:347]: Considering mentioning modifier-keys in a separate paragraph for simplicity and clarity.

For example:

```
Mouse buttons can be combined with modifier-keys (shift, super, control and alt), for example:

    <mousebind button="A-Right" action="Press">

```

**Comment by johanmalm** [comment]
Ready for merge?

**Comment by Consolatis** [comment]
> Ready for merge?

I think your statement about the new section isn't yet addressed, we might also want to rename and squash the commits.

**Comment by Consolatis** [comment]
Squashed the commits and change the commit title.

Thanks for the PR, merging once the CI ran through :)

**Pull request merged**